README.md

```diff
@@ -83,3 +71,46 @@ $ mypy -p opensquirrel
 ```shell
 $ pytest
 ```
+
+## Dev Container
+
+### PyCharm
+
+Open the Dev Container configuration file located at `.devcontainer/devcontainer.json` in the PyCharm editor. On the left of the editor, in the gutter, you should see the Docker icon: <img src="docs/_static/devcontainer_docker_icon.png" width="20" height="20"> or <img src="docs/_static/devcontainer_docker_icon_2.png" width="20" height="20">. Upon clicking on the Docker icon, the following options appear (depending on the PyCharm version used): **Create Dev Container and Mount Sources...** and **Show Dev Containers**. If no Dev Containers have been created previously, select the former option and proceed to create a Dev Container, otherwise select the latter option to list the existing Dev Containers.
```

Comment:
Just an advise that you can just ignore.
But there is the possibility to keep the lines in the raw file short.
Without affecting the output, how the file is displayed.
With the benefit to be more readable for the maintainer (this is just an opinion).
For example:

```
Open the Dev Container configuration file located at `.devcontainer/devcontainer.json` in the PyCharm editor.
On the left of the editor, in the gutter, you should see the Docker icon:
<img src="docs/_static/devcontainer_docker_icon.png" width="20" height="20"> or
<img src="docs/_static/devcontainer_docker_icon_2.png" width="20" height="20">.
Upon clicking on the Docker icon, the following options appear (depending on the PyCharm version used):
**Create Dev Container and Mount Sources...** and **Show Dev Containers**.
If no Dev Containers have been created previously,
select the former option and proceed to create a Dev Container,
otherwise select the latter option to list the existing Dev Containers.
```

Do not just document.
Write poems instead.
Haha!

Reply:
Good comment, I'll adjust the text and keep to this in the future :)